packages/fast-components-react-msft/src/badge/badge.schema.json

```diff
@@ -5,15 +5,9 @@
     "type": "object",
     "id": "badge",
     "properties": {
-        "appearance": {
-            "title": "Appearance",
-            "type": "string",
-            "default": "lowlight",
-            "enum": [
-                "lowlight",
-                "highlight",
-                "accent"
-            ]
+        "filled": {
+            "title": "Backplate",
```

Comment:
I have a bit of a disconnect between having a prop named "filled" and a title of "backplate". Do we care that the API is not accurately represented for design and developers here? If a designer mentions the backplate prop, will that cause confusion or require "special knowledge" that these map 1:1?

Reply:
@nicholasrice suggested `filled`, but in the editor view `backplate` made more sense to me. I understand the 1:1 concern.

My vote would be for `backplate` over `filled` - `filled` makes me ask "Filled what?"

Reply:
Pinging @nicholasrice who proposed `filled` - my assumption is that it maps to the v4 color work being done, but he can advise on the prop naming.

Reply:
v4 color work uses the term "fill" to describe recipes that should create component "backplates".

I think our use of the terms are relativly interchangable, but "fill" does have a long history in design vernacular to describe filling a shape with color. I'm uncertain how broadly "backplate" is used, but I have personally never seen it in design tools.

Reply:
@bheston @Falkicon My have thoughts here - I'm kinda impartial aside from that this name maps to *current* color recipe terms (which could change if we want)

Reply:
I don't think we have a precedence for something quite like this yet, but it's similar to the appearance property we have on button. One potential option would be to flip it. I think the default state for a badge should have a backplate, but understand that News has a particular usage of it where it's essentially just a single line of short text. Close to a typical badge, but questionable. Between the two I would suggest "filled" as well since we internally refer to things as backplates, but don't publish that anywhere. Another option that might be more descriptive would be something like "Contained" or "Framed", which adds the implication of the padding adjustment that's a key portion of the filled background. Conceivably you could have a "contained" badge with transparent fill, which would be like our lightweight button.

So, my suggestion would be "Filled" (default to true) or "Contained".

Reply:
I'm good with both changing to `filled` and defaulting to true. Will update.